GameData/SPEngine/Families/J.cfg

```diff
@@ -30,12 +30,12 @@ SPEFamily:NEEDS[RealFuels]
 			name = ElectricCharge
 			amount = 0.500
 		}
-		maxIgnitions = 2
+		maxIgnitions = 3
```

Comment:
???

Reply:
The first avalible configuration currently in game also has three ignitions, this change simply brings them both to the same number of ignitions.